pkg/kapp/cmd/core/deps_factory.go

```diff
@@ -17,6 +17,7 @@ import (
 
 type DepsFactory interface {
 	DynamicClient() (dynamic.Interface, error)
+	MutedDynamicClient() (dynamic.Interface, error)
```

Comment:
Let's not call it `MutetdDynamicClient` as we are planning to use it for all the internal(internal to kapp like list) client-go calls.

Reply:
Something like `InternalDynamicClient` ? Do you have any other suggestions ?